static/styles/portico/billing.scss

```diff
@@ -273,12 +273,3 @@
         display: none;
     }
 }
-
-input[type="number"]::-webkit-outer-spin-button,
-input[type="number"]::-webkit-inner-spin-button {
-    -webkit-appearance: none;
-    margin: 0;
-}
-input[type="number"] {
-    -moz-appearance: textfield;
-}
```

Comment:
The spin buttons are kinda ugly here; I think it'd be better to not remove this.  @hackerkid may remember the context more.

Reply:
Reordered the commits to separate this question to the end.

Reply:
Yes. I added those two styles two hide the buttons in Chrome and Firefox respectively. The spin buttons don't look good.

![Screenshot from 2020-08-08 13-28-03](https://user-images.githubusercontent.com/7190633/89705461-37e23d80-d97b-11ea-9366-d2cae8ce196f.png)
![Screenshot from 2020-08-08 13-27-34](https://user-images.githubusercontent.com/7190633/89705462-39136a80-d97b-11ea-83eb-ed88ff54931e.png)